README.rst

```diff
@@ -615,3 +629,41 @@ Stats
 
 Stats from python-zyte-api_ are exposed as Scrapy stats with the
 ``scrapy-zyte-api`` prefix.
+
+
+Request fingerprinting
+======================
+
+If you set the ``REQUEST_FINGERPRINTER_CLASS`` setting to
+``"scrapy_zyte_api.ScrapyZyteAPIRequestFingerprinter"``, `request
+fingerprinting <https://docs.scrapy.org/en/latest/topics/request-response.html#request-fingerprints>`_
+behaves as follows:
+
+-   Requests sent through Zyte API get a fingerprint based on the following
+    Zyte API parameters:
+
+    -   ``url`` (`canonicalized <https://w3lib.readthedocs.io/en/latest/w3lib.html#w3lib.url.canonicalize_url>`)
+
+    -   Request attribute parameters (``httpRequestBody``,
+        ``httpRequestMethod``)
+
+    -   Output parameters (``browserHtml``, ``httpResponseBody``,
+        ``httpResponseHeaders``, ``screenshot``)
```

Comment:
```suggestion
        ``httpResponseHeaders``, ``screenshot``, etc.)
```

Lots more to come next year with Automated Extraction.

Reply:
Probably best to append them later. We can then figure out how to refer to them, i.e. we may not want to list them all here.